tests/common/osutil/test_ubuntu.py

```diff
@@ -14,9 +14,12 @@
 #
 # Requires Python 2.6+ and Openssl 1.0+
 #
+
+import unittest
```

Comment:
Is this from any legacy code based on `unittest`? 

Also, as unittest is used in many places, and gets imported from `from tests.protocol.mockwiredata import *`, do you feel we should import `unittest` in `test.tools` itself?

Reply:
A lot of our test modules have this at the end of the file:
```
if __name__ == '__main__':
    unittest.main()
```
I didn't want to change this in this PR, since I'm not sure why it's even there. I'm guessing to allow running the tests even if the `nose` requirement is not satisfied.

I removed `from tests.protocol.mockwiredata import *` throughout specifically to prevent this type of shadow importing, so `unittest` needs to be explicitly imported. We can take the refactoring of it to `test.tools` along with the other modules Norberto mentioned in the next PR, I agree with the suggestion.